strings/src/main/res/values/strings.xml

```diff
@@ -1217,4 +1218,22 @@
     <string name="close_snackbar">Close snackbar</string>
 
     <string name="edit_finalized_form_warning">In later releases, you will not be able to edit finalized forms. Save forms as draft to edit them later.\n\nYou can check for errors in a draft form by tapping the three dots (⋮) and then Check for errors.</string>
+
+    <!-- Label for option that finalizes all the forms the user is currently looking at -->
+    <string name="finalize_all_forms">Finalize all forms</string>
+
+    <!-- Message shown when finalizing all forms has succeeded and how many totally have been finalized -->
+    <plurals name="bulk_finalize_success">
+        <item quantity="one">Success! %d form finalized.</item>
+        <item quantity="other">Success! %d forms finalized.</item>
+    </plurals>
+
+    <!-- Message shown when no forms finalized successfully -->
+    <plurals name="bulk_finalize_failure">
+        <item quantity="one">%d form has an error. Address issues before finalizing all forms.</item>
+        <item quantity="other">%d forms have errors. Address issues before finalizing all forms.</item>
+    </plurals>
+
+    <!-- Message shown when some forms finalize successfully and others fail -->
+    <string name="bulk_finalize_partial_success">%d forms finalized. %d forms have errors. Address issues before finalizing all forms.</string>
```

Comment:
I haven't found a nice way to make this a quantity string as it has two separate quantities (forms that finalised and those that didn't). We could create two strings and then combine them, but that doesn't feel worth it to me. I'm basing this on an assumption that other languages also don't have trouble with using plurals for singular in terms of the sentence making sense vs it being technically correct.